pkg/terminal/command.go

```diff
@@ -1063,10 +1068,49 @@ func setBreakpoint(t *Term, ctx callContext, tracepoint bool, argstr string) err
 	return nil
 }
 
+func setExecutionPoint(t *Term, ctx callContext, argstr string) error {
+	args := strings.SplitN(argstr, " ", 2)
+
+	requestedEp := &api.ExecutionPoint{}
+	locspec := ""
+	if len(args) != 1 {
+		return fmt.Errorf("address required")
+	} else {
+		locspec = argstr
+	}
+
+	locs, err := t.client.FindLocation(ctx.Scope, locspec)
+	if err != nil {
+		locspec = argstr
+		var err2 error
+		locs, err2 = t.client.FindLocation(ctx.Scope, locspec)
+		if err2 != nil {
+			return err
+		}
+	}
+	for _, loc := range locs {
```

Comment:
Jumping multiple times is idempotent to just doing the last jump, this should either reject the case where there are multiple locations, jump to the first returned location or jump to the last returned location (I think for now the appropriate thing is rejecting).